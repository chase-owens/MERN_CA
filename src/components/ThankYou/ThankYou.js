import React from "react";
import Typography from "@material-ui/core/Typography";
import { theme } from "../../styles/theme";
import ContactNav from "../ContactNav/ContactNav";
import { isMobile } from "react-device-detect";

const ThankYou = () => {
  return (
    <div>
      <ContactNav />
      <Typography
        style={{
          fontSize: isMobile ? "3.8em" : null,
          paddingTop: 20,
          fontSize: "2em",
        }}
        align="center"
        paragraph
        variant="h1"
      >
        Thank you for contacting us.
      </Typography>
      <Typography
        style={{
          fontSize: isMobile ? "3.8em" : null,
          fontSize: "2em",
        }}
        align="center"
        paragraph
        variant="h3"
      >
        You'll receive a reply from us within 24 hours.
      </Typography>
    </div>
  );
};

export default ThankYou;
