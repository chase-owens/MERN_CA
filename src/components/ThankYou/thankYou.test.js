import React from 'react';
import ThankYou from './ThankYou';
import { shallow } from 'enzyme';

describe('ThankYou', () => {
  it('renders without crashing', () => {
    const thankYou = shallow(<ThankYou />);
    expect(thankYou).toMatchSnapshot();
  });
});
