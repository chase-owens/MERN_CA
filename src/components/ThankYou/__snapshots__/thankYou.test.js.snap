// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThankYou renders without crashing 1`] = `
<div>
  <Connect(ContactNav) />
  <WithStyles(Typography)
    align="center"
    paragraph={true}
    style={
      Object {
        "fontSize": "2em",
        "paddingTop": 20,
      }
    }
    variant="display4"
  >
    Thank you for contacting us.
  </WithStyles(Typography)>
  <WithStyles(Typography)
    align="center"
    paragraph={true}
    style={
      Object {
        "fontSize": "2em",
      }
    }
    variant="display4"
  >
    You'll receive a reply from us within 24 hours.
  </WithStyles(Typography)>
</div>
`;
